subsys/bluetooth/controller/ll_sw/nordic/lll/lll_test.c

```diff
@@ -482,6 +460,24 @@ static bool check_rx_cte(bool cte_ready)
 }
 #endif /* CONFIG_BT_CTLR_DF_CTE_RX */
 
+static uint32_t calculate_tifs(uint8_t len)
+{
+	uint32_t interval;
+	uint32_t transmit_time;
+
+#if defined(CONFIG_BT_CTLR_DF_CTE_TX)
+	/* Include additional byte for the CTEInfo field and CTE length in microseconds. */
+	transmit_time = PDU_US((test_cte_len > 0) ? (len + 1) : len, 0, test_phy, test_phy_flags) +
+			CTE_LEN_US(test_cte_len);
+#else
+	transmit_time = PDU_US(len, 0, test_phy, test_phy_flags);
+#endif /* CONFIG_BT_CTLR_DF_CTE_TX */
+
+	interval = ceiling_fraction((transmit_time + 249), SCAN_INT_UNIT_US) * SCAN_INT_UNIT_US;
```

Comment:
It would be good to document the magic number, though I know it comes from the original code.

Reply:
I like this idea. Done